<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IPConfigSucessDialog</class>
 <widget class="QDialog" name="IPConfigSucessDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>关于</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/logo.png</normaloff>images/logo.png</iconset>
  </property>
  <widget class="QPushButton" name="pushButtonConfig">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>102</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>子网掩码:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>51</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>IP:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_IP">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>51</y>
     <width>300</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_submask">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>102</y>
     <width>300</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>255.255.255.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_netgate">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>300</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>255.255.255.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>157</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>网关:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConfigMannual">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>手工配置</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
